src/pages/settings/AddSecondaryLoginPage.js

```diff
@@ -107,7 +107,7 @@ class AddSecondaryLoginPage extends Component {
                     onBackButtonPress={() => Navigation.navigate(ROUTES.SETTINGS_PROFILE)}
                     onCloseButtonPress={() => Navigation.dismissModal()}
                 />
-                <ScrollView style={styles.flex1} contentContainerStyle={styles.p5}>
+                <ScrollView style={styles.flex1} contentContainerStyle={styles.p5} keyboardShouldPersistTaps="handled">
```

Comment:
Why is this necessary?

Reply:
I believe I've seen this before when the intention is to allow someone to tap outside of an input so that the input will blur and the keyboard will dismiss. This was the case for the old Login form before I changed it in this PR: https://github.com/Expensify/App/pull/12848